util/gem5_bridge/src/gem5_bridge.h

```diff
@@ -0,0 +1,46 @@
+/* SPDX-License-Identifier: BSD AND GPL-2.0+ */
```

Comment:
These licenses seem a bit strange to me. It seems like it should be BSD-3-Clause (there isn't a plain BSD license in the SPDX license list). The GPL license should probably be GPL-2.0-only to match Linux's license (you'll sometimes see GPL-2.0 as well, but that's deprecated).

It's also unclear to me if the intention is that the user has the choice to apply the GPL license or the BSD license, or if both licenses always apply. The SPDX 'AND' operator specifies the latter. If the intention is to allow either license, the correct identifier would be 'BSD-3-Clause OR GPL-2.0-Only'.

Reply:
Fixed, thanks!